app/serializers/regional_serializer.rb

```diff
@@ -80,6 +80,14 @@ def filters
         options: unique_license,
         sortButtons: false,
         type: "search"
+      },
+      {
+        name: "language",
```

Comment:
I think the language filter is just for the global page.

Reply:
( @lucacug )

Reply:
@WKocur That should be fixed now